geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -5325,7 +5325,7 @@ public void basicBridgeClientUpdate(DistributedMember serverId, Object key, Obje
     // If the marker has been processed, process this put event normally;
     // otherwise, this event occurred in the past and has been stored for a
     // durable client. In this case, just invoke the put callbacks.
-    if (processedMarker) {
+    if (processedMarker || hasRegisterInterestsWithResultPolicy(true, InterestResultPolicy.NONE)) {
```

Comment:
This is the other critical part of the change to resolve this bug.  Thanks to @boglesby for the code for hasRegisterInterestsWithResultPolicy